File: Results/DER/Translation/translation_output/no_test/magicoder/codenet/python/java/p03250.java
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following python code to java and enclose your solution inside ```java```:
```
abc = list(map(int, input().split()))
abc.sort()
print(int(str(abc[2]) + str(abc[1])) + abc[0])
```

@@ Response
```java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int[] abc = new int[input.length];
        for (int i = 0; i < input.length; i++) {
            abc[i] = Integer.parseInt(input[i]);
        }
        Arrays.sort(abc);
        System.out.println(Integer.parseInt(String.valueOf(abc[2]) + String.valueOf(abc[1])) + abc[0]);
    }
}
```

This Java code does the same thing as the Python code you provided. It reads a line of input, splits it into an array of strings, converts each string to an integer, sorts the array, and then concatenates the last two elements of the sorted array (converted back to strings) with the first element and prints the result.<｜end▁of▁sentence｜>

File: Results/DER/Translation/translation_output/w_test/deepseekcoder/codenet/java/python-sanitized/p03250.py
def main():
    a, b, c = map(int, input().split())
    if a >= b and a >= c:
        if b >= c:
            print(int(str(a) + str(b)) + c)
        else:
            print(int(str(a) + str(c)) + b)
    elif b >= a and b >= c:
        if a >= c:
            print(int(str(b) + str(a)) + c)
        else:
            print(int(str(b) + str(c)) + a)
    else:
        if a >= b:
            print(int(str(c) + str(a)) + b)
        else:
            print(int(str(c) + str(b)) + a)

if __name__ == "__main__":
    main()